gradle/setup.gradle

```diff
@@ -35,12 +35,12 @@ configure(subprojects) { p ->
 	}
 
 	task sourcesJar(type: Jar) {
-		classifier = 'sources'
+		archiveClassifier.set('sources')
 		from sourceSets.main.allSource
 	}
 
 	task javadocJar(type: Jar) {
-		classifier = 'javadoc'
+	  archiveClassifier.set('javadoc')
```

Comment:
The indentation strikes again :D We should at least make it consistent in this file
<img width="447" alt="Screenshot 2019-10-22 at 5 29 25 PM" src="https://user-images.githubusercontent.com/1050762/67302671-84928000-f4f1-11e9-9c08-ea74d7e4437e.png">

Reply:
yeah, convert spaces to tabs in IntelliJ does not work on this particular file for some reason :( probably due to the heavy mixing of tabs and spaces

Reply:
fixed